types/requests.ts

```diff
@@ -31,6 +31,12 @@ import {
 export interface BaseParams {
   safetySettings?: SafetySetting[];
   generationConfig?: GenerationConfig;
+  // Presence penalty applied to the next token's logprobs if the token has
```

Comment:
In order to show up in the documentation, comments need to be in block format, e.g.

```
/**
 * Presence penalty applied to the next token's logprobs if the token has
 * already been seen in the response.
 */
```

For reference: https://api-extractor.com/pages/tsdoc/doc_comment_syntax/